docs/_static/scroll_sidebar.js

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+
+"use strict";
+
+window.addEventListener("DOMContentLoaded", main);
+
+function main() {
+    const currentSection = document.querySelector("#sidebar li.current");
+
+    if (currentSection) {
+        currentSection.scrollIntoView();
+    }
```

Comment:
This could be moved to `sidebar.js` or `custom.js`.

Reply:
sidebar.js is probably better